CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
+
+* Bug Fixes:
+  * Fix an [issue](https://github.com/azure-networking/cilium-private/pull/465)) in [Azure CNI Powered by Cilium](https://learn.microsoft.com/azure/aks/azure-cni-powered-by-cilium), if the DNS request timeout is set to below 2 seconds, the high-scale scenarios may cause drops due to duplicate request ID.
+  * Fix an issue where [load balancer](https://learn.microsoft.com/azure/aks/load-balancer-standard) tags were not updated after a managed cluster tag update. Load balancer tags now correctly reflect the latest state.
+  * Fix an [issue](https://github.com/cilium/cilium/issues/38210) in Cilium v1.17 where a deadlock causing pods unable to start. 
+
+* Behavior Changes:
+  * `aksmanagedap` is blocked as a reserved name for use of creating agent pool.
+  * `linuxutil plugin` is temporarily disabled for [Retina Basic and ACNS](https://retina.sh/docs/Metrics/plugins/Linux/linuxutil) for it is causing memory leaks that leads to Retina pods OOMKill.
+  * ACNS configmaps (`cilium`, `retina`, `hubble`) now auto‑format cluster names to satisfy Cilium 1.17 rules:≤ 32 chars, lowercase alphanumeric characters and dashes, no leading/trailing dashes. See this [link](https://github.com/cilium/cilium/commit/b1f10bcb26d71f046d8ad108d5f4b9e42aa5d486) for details.
+  * The `defaultConfig.gatewayTopology` field is now included in the Istio add-on `MeshConfig` AllowList as an unsupported field. For more details, see the [Istio MeshConfig documentation](https://learn.microsoft.com/azure/aks/istio-meshconfig#allowed-supported-and-blocked-meshconfig-values).
+  * You can now disable [NodeAutoProvisioning](https://learn.microsoft.com/azure/aks/node-autoprovision) if meet these criteria: 1. The cluster must have 0 NAP-managed nodes. 2. Every existing karpenter.sh/NodePool must have its spec.limits.cpu set to 0.
+  * [Disabling kube-proxy](https://learn.microsoft.com/azure/aks/configure-kube-proxy) no longer requires the AFEC flag in bring-your-own (BYO) CNI scenarios.
+  * [ExtensionManager](https://learn.microsoft.com/azure/azure-arc/kubernetes/conceptual-extensions) is moved from kube-system to internal AKS infrastructure. Clusters with extensions in canary no longer require any of the outbound networking rules documented [here](https://learn.microsoft.com/azure/aks/outbound-rules-control-egress#cluster-extensions). Note that some extensions such as Flux still require a subset of these outbound rules.
+
+* Component Updates:
+  * Fleet networking components updated to v0.39 from v0.38 to fix CVE.
```

Comment:
@sjwaight, @ryanzhang-oss is this component visible on the clusters (afaik no) or is this component open sourced? if no on both questions, we need to remove this line. if yes to second question, then we need to add a link to release notes for v0.39

Reply:
oh, it's open sourced, the release note is https://github.com/Azure/fleet-networking/releases/tag/v0.3.9